python/ray/autoscaler/kubernetes/operator_configs/operator.yaml

```diff
@@ -0,0 +1,50 @@
+operator_role:
```

Comment:
Is this necessary for making the ARM build work?

Reply:
No it is not, sorry this is an unrelated K8s change, I'll move it over.